index.bs

```diff
@@ -913,7 +915,7 @@ The <dfn method for=CookieStoreManager>unsubscribe(|subscriptions|)</dfn> method
 ## The {{ServiceWorkerRegistration}} interface ## {#ServiceWorkerRegistration}
 <!-- ============================================================ -->
 
-The {{ServiceWorkerRegistration}} interface is extended to give access to a {{CookieStoreManager}}.
+The {{ServiceWorkerRegistration}} interface is extended to give access to a {{CookieStoreManager}} via {{cookies}} which provides the interface for subscribing to cookie changes.
 
 <xmp class=idl>
 [Exposed=(ServiceWorker,Window)]
```

Comment:
kind of separate to this PR, but cookies should probably be [SameObject]. And the cookies attribute should normatively declare what it returns somewhere, i.e. something like this:

```
Each {{ServiceWorkerRegistration}} has an associated {{CookieStore}} object.
The {{CookieStore}}'s [=CookieStore/registration=] is equal to the {{ServiceWorkerRegistration}}'s [=ServiceWorkerRegistration/service worker registration=].

The {{ServiceWorkerRegistration/cookies}} [=getter steps=] are to return [=this=]'s associated {{CookieStore}} object.
```

Reply:
Thanks for the guidance! I made the change to SameObject and added the normative bit below. Made some changes to get the links to work, but let me know if I've missed anything.

Reply:
Huh, I see. Technically it really should be [=ServiceWorkerRegistration/service worker registration=], but I guess that term isn't exported by the service worker spec, so what you have now seems good enough.
(as background, [=service worker registration=] would link to https://w3c.github.io/ServiceWorker/#dfn-service-worker-registration, while what we really want here is https://w3c.github.io/ServiceWorker/#serviceworkerregistration-service-worker-registration, but as said that term isn't exported. It's not like it isn't clear what we're talking about so the current text seems fine to me).

Reply:
I see, just out of curiosity how can you tell if a term has been exported or not?